<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PolarChartDialog</class>
 <widget class="QDialog" name="PolarChartDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Polart Chart Setup</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Display mode</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Frequency:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="displayModeFreq">
              <item>
               <property name="text">
                <string>Show complete traces</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Limit to current span</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="displayFreqOverride">
            <property name="text">
             <string>Override visible frequency range</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Start:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="SIUnitEdit" name="displayStartFreq"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Stop:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="SIUnitEdit" name="displayStopFreq"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Î“</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="displayModeRefl">
              <item>
               <property name="text">
                <string>Show complete traces</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Limit to visible area</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Zoom</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Factor:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="SIUnitEdit" name="zoomFactor"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Offset X axis:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="SIUnitEdit" name="offsetXaxis"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Offset Y axis:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="SIUnitEdit" name="offsetYaxis"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SIUnitEdit</class>
   <extends>QLineEdit</extends>
   <header>CustomWidgets/siunitedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PolarChartDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PolarChartDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
